File: sum_list/sum_list.py

# class Node:
#   def __init__(self, val):
#     self.val = val
#     self.next = None
​
def sum_list(head):
  sum = 0
  while head is not None:
    sum += head.val
    head = head.next
    
  return sum
​
const sumList = (head) => {

File: sum_list/sum_list.js

class Node {
  constructor(val) {
    this.val = val;
    this.next = null;
  }
}
​
​
const sumList = (head) => {
  let sum = 0;
  let current = head;
  while (current !== null) {
    sum += current.val;
    current = current.next;
  }
  
  return sum
};
​
​
​
​
​
​
​
​
​
​
​
​
​
​
​
​
​